ndsl/grid/generation.py

```diff
@@ -237,7 +237,7 @@ def __init__(
         dy_const: float = 1000.0,
         deglat: float = 15.0,
         extdgrid: bool = False,
-        eta_file: str = "None",
+        eta_file: str = "",
```

Comment:
Default to `None`

Reply:
I had tried that initially, but the linting crashed.